content/en/apps/tutorials/application-tests.md

```diff
@@ -0,0 +1,600 @@
+
+Use [harness.getContactSummary()](https://docs.communityhealthtoolkit.org/cht-conf-test-harness/Harness.html#getContactSummary) method to get the [ContactSummary ](https://docs.communityhealthtoolkit.org/cht-conf-test-harness/global.html#ContactSummary)object, which has these properties: `fields`, `cards`, and `context`.
+
+To test the contact summary fields added in the [previous tutorial]({{< ref "apps/tutorials/contact-summary#3-export-fields" >}}), use the following test case:
+
+```js {linenos=table}
+const contactSummary = await harness.getContactSummary();
+expect(contactSummary.fields).to.have.property('length', 5);
+expect(contactSummary.fields.filter(f => f.filter !== 'lineage')).to.deep.equal(
+    [
+      { label: 'patient_id', value: 'patient_id', width: 4 },
+      { label: 'contact.age', value: '1970-07-09', width: 4, filter: 'age' },
+      { label: 'contact.sex', value: 'contact.sex.female', translate: true, width: 4 },
+      { label: 'person.field.phone', value: undefined, width: 4}
+    ]
+  );
+```
+
+Here, the contact summary being tested represents the contact that is being "acted on" or the "subject of the test". To learn more about this, look at the `subject` property [here](https://docs.communityhealthtoolkit.org/cht-conf-test-harness/global.html#HarnessInputs).
+
+Similarly, you can test the condition cards too. Here is an example for testing the assessment condition card added in this [tutorial]({{< ref "apps/tutorials/condition-cards#2-define-cards-and-add-a-condition-card-object" >}}):
+
+```js {linenos=table}:
+// Load the assessment form and fill in 'yes' on the first page and '7' on the second page
+const result = await harness.fillForm('assessment', ['yes'], ['7']);
+
+// Verify that the form successfully got submitted
+expect(result.errors).to.be.empty;
+
+const contactSummary = await harness.getContactSummary();
+
+// Verify that there is one card
+expect(contactSummary.cards).to.have.property('length', 1);
+
+// Verify the fields on the card
+expect(contactSummary.cards[0].fields).to.deep.equal(
+  [
+    {
+      label: 'contact.profile.most_recent_assessment.date',
+      value: result.report.reported_date,
+      width: 6,
+      filter: 'simpleDate'
+    },
+    { label: 'contact.profile.cough', value: 'yes', width: 6 },
+  ]
+);
+```
+If you  follow [this code sample]({{< ref "apps/reference/contact-page#code-samples" >}}) to create the pregnancy condition card, the pregnancy context can be tested this way:
+```js {linenos=table}
+const summaryContext = harness.getContactSummary().context;
+expect(summaryContext).to.include({
+  pregnant: true
+});
+```
+---
+
+### Testing Tasks
+
+<table>
```

Comment:
Similar to above comments on table, please refer to: https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#use-markdown-notation

Reply:
Converted. Thanks!